packages/run-protocol/src/vaultFactory/vaultFactory.js

```diff
@@ -199,9 +214,20 @@ export const start = async (zcf, privateArgs) => {
     return vaultParamManagers.get(paramDesc.collateralBrand).getParams();
   };
 
+  const getCollateralManager = brandIn => {
+    assert(
+      collateralTypes.has(brandIn),
+      X`Not a supported collateral type ${brandIn}`,
+    );
+    /** @type {VaultManager} */
+    return collateralTypes.get(brandIn).getPublicFacet();
+  };
+
   const publicFacet = Far('vaultFactory public facet', {
+    getCollateralManager,
     /** @deprecated use makeVaultInvitation instead */
     makeLoanInvitation: makeVaultInvitation,
+    /** @deprecated use getCollateralManager and then makeVaultInvitation instead */
```

Comment:
Why is this better than taking a collateral type on `makeVaultInvitation`?
```
  // TODO add a `collateralBrand` argument to makeVaultInvitation`
  /** Make a loan in the vaultManager based on the collateral type. */
  const makeVaultInvitation = () => {
    return zcf.makeInvitation(makeVaultHook, 'MakeVault');
  };
```

If that is deprecated then it should have `@deprecated` (which I think will flow here but no harm in repeating)

Reply:
I think it's better because we can then add `getNotifier` and other operations on a per-collateral basis. It also enables collateral-specific extensions (if e.g., for stable token collateral, liquidity token collateral, etc.